ATGMediaBrowser/MediaBrowser/MediaBrowserViewController.swift

```diff
@@ -158,6 +158,24 @@ public class MediaBrowserViewController: UIViewController {
         case nextToPrevious
     }
 
+    /**
+     Struct to hold support for customize title style
+
+     ```
+     font
+     textColor
+     ```
+    */
+    public struct TitleStyle {
+        var font: UIFont
+        var textColor: UIColor
+
+        init?(font: UIFont = UIFont.preferredFont(forTextStyle: .subheadline), textColor: UIColor = .white) {
```

Comment:
No need to make it failable initializer. We are not returning nil for any cases inside the init.

Reply:
Newline after opening brace please.